src/main/java/pl/javastart/task/Televisor.java

```diff
@@ -0,0 +1,18 @@
+package pl.javastart.task;
+
+public class Televisor {
+    private boolean isOn;
+
+    public void turnOn() {
+       isOn = true;
```

Comment:
checkstyle: wcięcie
Checkstyle zauważył w tej linii błąd. To, co znalazł możesz podejrzeć w pull requeście z zakładce "Files changed". Polecam zainstalować sobie dodatek do intellij, żeby na bieżąco informował. Instrukcję znajdziesz tutaj. W instrukcji jest też informacja jak ustawić dla wszystkich nowych projektów. https://bootcamp.javastart.pl/strona/jakosc-kodu